CHANGELOG.md

```diff
@@ -32,7 +32,7 @@
 
 ### Changes
 
-- Introduce tail-based tracing framework with [minitrace](https://github.com/tikv/minitrace-rust). [Tail-based Tracing Example](https://github.com/MrCroxx/foyer/tree/main/examples/tail_based_tracing.rs).
```

Comment:
The change log of the previous version shouldn't be modified. Let me fix it for you.